librz/bin/format/elf/elf_info.c

```diff
@@ -989,11 +990,11 @@ static bool get_versym_entry_sdb_from_verneed(ELFOBJ *bin, Sdb *sdb, const char
 				continue;
 			}
 
-			if (vernaux_entry.vna_name > bin->strtab_size) {
+			char tmp[ELF_STRING_LENGTH];
+			if (!Elf_(rz_bin_elf_strtab_cpy)(bin->strtab, tmp, vernaux_entry.vna_name)) {
 				return false;
 			}
-
-			const char *value = sdb_fmt("%u (%s)", versym & VERSYM_VERSION, bin->strtab + vernaux_entry.vna_name);
+			const char *value = sdb_fmt("%u (%s)", versym & VERSYM_VERSION, tmp);
```

Comment:
Sdb entries doesn't seems to be used. In the future it should be deleted.

Reply:
If you want, I can delete sdb usage in the next PR

Reply:
@08A they are used by the `pf.` commands when you load the binary in `-nn` mode.

Reply:
I don't know how pf work, but in theory those information can be recreated with the bin->* information right?

Reply:
It works this way:
- Parses KV with `*.cparse` name into corresponding types like enums, for example
- Parses the rest as `pf` formats.
- When you use `pf.` command it uses information from both.

I recommend not to touch this part, since it will affect my RzTypes refactoring as well. I agree that this is suboptimal, but better to wait for a while.

Reply:
I will fix sdb_fmt usage in a following PR